innative/parse.cpp

```diff
@@ -331,11 +335,20 @@ IN_ERROR innative::ParseInstruction(Stream& s, Instruction& ins, const Environme
   case OP_i64_store8:
   case OP_i64_store16:
   case OP_i64_store32:
+    // Alignment and memidx bit flag
     ins.immediates[0]._varuint32 = s.ReadVarUInt32(err);
 
+    // Offset
     if(err >= 0)
       ins.immediates[1]._varuptr = s.ReadVarUInt32(err); // Currently 32-bit because all memories are 32-bit
 
+    // If bit 6 is set, read a memidx value
+    if(err >= 0 && (ins.immediates[0]._varuint32 & 0b1000000))
```

Comment:
Should this access really be _varuint32? Please check or confirm the types of memidx values.